_posts/2017-11-23-ABRT-in-container.markdown

```diff
@@ -0,0 +1,224 @@
+---
+layout: post
+title:  "ABRT in openshift container"
+date:   2017-11-24 12:18:21 +0100
+author: Julius Milan <jmilan@redhat.com>
+visible: 1
+categories: abrt openshift container deploy
+---
+ABRT is able to run inside an openshift resp. docker container and catch certain subset of
+problems occured on the host on which the container is running including all other containers
+running on the same host.
+
+This setup has two limitation:
+* Catches just problems producing coredumps, i.e. C and C++ crashes.
+* Catches just on the host where abrt container is running.
+
+# Deploy openshift ABRT container
+First we need to stop abrt daemon on the host if its running:
+```
+sudo systemctl stop abrtd
+```
+Then, assuming the openshift cluster is running, log in as administrator and create a project
+named abrt:
+```
+oc login -u system:admin
+oc new-project abrt
+```
+Then, use a following template abrt-template.yaml:
```

Comment:
What about adding the abrt-template.yaml as a file? And then add a link to it here so there is an option to download it with wget?